Makefile

```diff
@@ -55,6 +56,7 @@ bootstrap: \
 	run \
 	migrate \
 	i18n-compile-back \
+	build-mails \
```

Comment:
At first bootstrap this task is failing. You must install first the packages. You should add another task installing the node packages and `build-mails` should depends on this task.